app/views/providers/partners/bank_accounts/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= page_template page_title: t(".h1-heading"), template: :basic, show_errors_for: @form do %>
+  <%= form_with(
+        model: @form,
+        url: providers_legal_aid_application_partners_bank_accounts_path(@legal_aid_application),
+        method: :patch,
+        local: true,
+      ) do |form| %>
+
+    <%= form.govuk_check_boxes_fieldset :savings_amount,
+                                        legend: { text: page_title, size: "xl", tag: "h1" },
+                                        hint: { text: t("generic.select_all_that_apply") },
+                                        classes: "govuk-!-padding-bottom-4" do %>
+      <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+        <div class="deselect-group" data-deselect-ctrl="#savings-amount-no-partner-account-selected-true-field">
+          <%= render partial: "shared/forms/revealing_checkbox/attribute",
```

Comment:
In AP-4143 I have removed the use of this partial from all the other capital question pages - not sure whether it makes sense to do the same here or if it's still needed?

Reply:
Let me take a look at your PR while reviewing and then re-assess this code :+1:

Reply:
This has been refactored and removed - can you take a look at the changes?